modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -698,6 +713,51 @@ Replaced by: `ALTER USER`.
 
 == Procedure descriptions
 
+[[procedure_cdc_current]]
+.cdc.current()
+[cols="<15s,<85"]
+|===
+| Description
+a|
+Fetch the current `changeId` for use with xref:reference/procedures.adoc#procedure_cdc_unfiltered_query[`cdc.unfiltered.query()`]
+
+For mor details, see link:{neo4j-docs-base-uri}/change-data-capture/{page-version}/introduction[Change Data Capture].
+| Signature
+m|cdc.current() :: STRING NOT NULL
+| Mode
+m|READ
+|===
+
+[[procedure_cdc_earliest]]
+.cdc.earliest()
+[cols="<15s,<85"]
+|===
+| Description
+a|
+Fetch the earliest available `changeId` for use with xref:reference/procedures.adoc#procedure_cdc_unfiltered_query[`cdc.unfiltered.query()`]
+
+For mor details, see link:{neo4j-docs-base-uri}/change-data-capture/{page-version}/introduction[Change Data Capture].
+| Signature
+m|cdc.earliest() :: STRING NOT NULL
+| Mode
+m|READ
+|===
+
+[[procedure_cdc_unfiltered_query]]
+.cdc.unfiltered.query()
+[cols="<15s,<85"]
+|===
+| Description
+a|
+Query the transaction log for database change events since from, optionally scoped to changes matching the selectors list.
+For mor details, see link:{neo4j-docs-base-uri}/change-data-capture/{page-version}/introduction[Change Data Capture].
+| Signature
+m|cdc.unfiltered.query(from = cdc.current() :: STRING NOT NULL, selectors = [] :: LIST OF MAP) :: (id :: STRING NOT NULL, txId :: INTEGER NOT NULL, seq :: INTEGER NOT NULL, metadata :: MAP NOT NULL, event :: MAP NOT NULL)
```

Comment:
`LIST OF MAP` is nowadays called `LIST<MAP>`

Reply:
No longer relevant after d721ad9